<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-17T10:29:23 IST" name="LeadAngelV1" finished-at="2024-09-17T10:30:28 IST" duration-ms="65048">
    <groups>
    </groups>
    <test started-at="2024-09-17T10:29:23 IST" name="LA_chrome" finished-at="2024-09-17T10:30:28 IST" duration-ms="65048">
      <class name="com.leadAngel.testCases.TC_Sanity_ObjectExplorer_Administration">
        <test-method is-config="true" signature="setup(java.lang.String, java.lang.String)[pri:0, instance:com.leadAngel.testCases.TC_Sanity_ObjectExplorer_Administration@464a4442]" started-at="2024-09-17T10:29:23 IST" name="setup" finished-at="2024-09-17T10:29:30 IST" duration-ms="7808" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="AdministrationTest()[pri:0, instance:com.leadAngel.testCases.TC_Sanity_ObjectExplorer_Administration@464a4442]" started-at="2024-09-17T10:29:30 IST" name="AdministrationTest" description="Administration_ObjectExplorer_Section" finished-at="2024-09-17T10:30:27 IST" duration-ms="56597" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.leadAngel.testCases.TC_Sanity_ObjectExplorer_Administration.AdministrationTest(TC_Sanity_ObjectExplorer_Administration.java:205)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AdministrationTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.leadAngel.testCases.TC_Sanity_ObjectExplorer_Administration@464a4442]" started-at="2024-09-17T10:30:27 IST" name="tearDown" finished-at="2024-09-17T10:30:28 IST" duration-ms="269" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.leadAngel.testCases.TC_Sanity_ObjectExplorer_Administration -->
    </test> <!-- LA_chrome -->
  </suite> <!-- LeadAngelV1 -->
</testng-results>
